wikipedia-templates.csl

```diff
@@ -146,7 +171,7 @@
   </macro>
   <macro name="pagination">
     <choose>
-      <if type="article-journal chapter">
+      <if type="article-journal chapter entry-encyclopedia">
```

Comment:
This needs match="any" (should have been in previous version too)

Reply:
Indeed. Thanks for picking that up. Fixed now.